spec/chapel_listing.tex

```diff
@@ -11,12 +11,12 @@
       label, lambda, let, local, locale,
       module,
       new, nil, noinit,
-      on, only, opaque, otherwise, out,
+      on, only, opaque, otherwise, out, owned, override,
```

Comment:
"override" should come before "owned" alphabetically.

Reply:
It looks like you've done that everywhere the two have been listed next to each other